fortio_main.go

```diff
@@ -85,6 +86,7 @@ var (
 	numThreadsFlag    = flag.Int("c", defaults.NumThreads, "Number of connections/goroutine/threads")
 	durationFlag      = flag.Duration("t", defaults.Duration, "How long to run the test or 0 to run until ^C")
 	percentilesFlag   = flag.String("p", "50,75,90,99,99.9", "List of pXX to calculate")
+	mergeFilesFlag    = flag.String("files", "", "List of json files to merge")
```

Comment:
how is the list specified ? (what is the separator) or does it work like proxiesFlags where you can have many -P ?

files is also a bit vague.

As this is just going to be a new command, like `fortio merge ...` then no need for flags just have that command and the rest of the args are the files (a flag would be some -include-file or such to read files from a file/stdin instead of command line)

You also probably want to make an "fmerge" like there is an fcurl and echosrv standalone options